src/main/kotlin/lotto/model/Money.kt

```diff
@@ -0,0 +1,21 @@
+package lotto.model
+
+data class Money(val amount: Int) {
+    fun dividedBy(price: Money): Double {
```

Comment:
코틀린에서는 정해진 이름의 함수를 오버로딩함으로써 표준 수학 연산자를 오버로딩할 수 있습니다. 아래의 글이 도움 될 거예요.
https://kotlinlang.org/docs/operator-overloading.html

```suggestion
    operator fun div(price: Money): Double {
```